components/Icon/Icon.jsx

```diff
@@ -106,6 +109,9 @@ const Icon = ({ name, skin, size, ...props }) => {
     visibility: Visibility,
     warning: Warning,
     home: Home,
+    accessible: Accessible,
```

Comment:
what do you think to sort icons by alphabetic?

Reply:
Agree. Done.